### PR TITLE
Make sure we grab all comments for github

### PR DESCRIPTION
**Pull request by orta**
in #32 I wondered if there were x amounts of comments being downloaded, looks like 30 are downloaded. It's definitely possible that a PR can get that many comments, so I think it's worth letting danger deal with all of them.

### CODE REVIEW

**Comment by orta** [comment]
I've added rubocop to `rake spec` too :+1:

**Comment by orta** [comment]
screen shot 2016-01-30 at 16 34 04
err, hrm, that did not work.

File: Rakefile [hunk 1]
```diff
@@ -1,10 +1,15 @@
 require "bundler/gem_tasks"
 require "rspec/core/rake_task"
 
-RSpec::Core::RakeTask.new(:spec)
+RSpec::Core::RakeTask.new(:specs)
 
 task default: :spec
 
+task :spec do
+    Rake::Task['specs'].invoke
```
Comment (KrauseFx) [inline Rakefile]: Should be 2 spaces, right?

File: lib/danger/request_sources/github.rb [hunk 2]
```diff
@@ -50,7 +50,7 @@ def update_pull_request!(warnings: nil, errors: nil, messages: nil)
         # Just remove the comment, if there's nothing to say.
         delete_old_comments!
       else
-        issues = client.issue_comments(ci_source.repo_slug, ci_source.pull_request_id)
+        issues = client.issue_comments(ci_source.repo_slug, ci_source.pull_request_id, auto_pagination: true)
```
Comment (KrauseFx) [inline lib/danger/request_sources/github.rb]: AFAIK it's possible to set this globally, so we don't have to do this for each request

**Comment by orta** [comment]
Alright, using the global setting for auto paginate, and fixed the `rake spec` doing all of ruby

**Comment by KrauseFx** [comment]
Looks good, and tests are passing :rocket:

**Comment by KrauseFx** [comment]
Shipping so fast :heart:

**Pull request merged**